ansible/roles/operator-pipeline/templates/openshift/triggers/operator-hosted-pipeline-trigger.yml

```diff
@@ -0,0 +1,138 @@
+---
+apiVersion: triggers.tekton.dev/v1alpha1
+kind: EventListener
+metadata:
+  name: operator-hosted-pipeline-github-listener
+spec:
+  serviceAccountName: pipeline
+  triggers:
+    - name: github-listener
+      interceptors:
+        - cel:
+            filter: >-
+              (
+                header.match("X-GitHub-Event", "pull_request")
+                && body.action in ["opened", "reopened", "synchronize"]
+                && body.pull_request.base.ref == "main"
+              )
+      bindings:
+        - ref: operator-hosted-pipeline-trigger-binding
+      template:
+        name: operator-hosted-pipeline-trigger-template
+
+---
+# To see the PR webhook payload, set the logLevel to debug in configmap config-logging-triggers
+# Other GitHub PR webhook payload example: https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#pull_request
+apiVersion: triggers.tekton.dev/v1alpha1
+kind: TriggerBinding
+metadata:
+  name: operator-hosted-pipeline-trigger-binding
+spec:
+  params:
+    - name: git_pr_branch
+      value: $(body.pull_request.head.ref)
+    - name: git_pr_title
+      value: $(body.pull_request.title)
+    - name: git_pr_url
+      value: $(body.pull_request.html_url)
+    - name: git_fork_url
+      value: $(body.pull_request.head.repo.html_url)
+    - name: git_repo_url
+      value: $(body.pull_request.base.repo.html_url)
+    - name: git_username
+      value: $(body.pull_request.head.user.login)
+    - name: pr_head_label
+      value: $(body.pull_request.head.label)
+    - name: pyxis_url
+      value: https://pyxis.engineering.redhat.com/
+    - name: preflight_min_version
+      value: "1.0"
+    - name: ci_min_version
+      value: "1.0"
```

Comment:
Please put a TODO or related comment that we need to revisit these values.

Reply:
These values should be configurable.